.github/workflows/acceptance.yml

```diff
@@ -143,7 +185,23 @@ jobs:
           name: frontend-dist
           path: dist.tar.gz
 
+  # It is necessary to update the name and matrix in tandem with the acceptance workflow
+  # to meet the required check
+  fake-acceptance:
+    if: needs.changes.outputs.backend_or_frontend != 'true'
+    needs: changes
+    name: acceptance
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.8.12]
+        instance: [0, 1, 2, 3]
```

Comment:
The matrix has to match so we can meet the "Required" checks requirement.

Reply:
Can we add this as a comment to the real job

Reply:
Added to the real workflows.